README.md

```diff
@@ -27,6 +27,53 @@ If you want it to be included in the final `client-jsdeps.js`, you can add the d
     jsDependencies +=
       "org.webjars" % "jquery" % "2.1.3" / "2.1.3/jquery.js"
 
+Using [Scala.js Bundler](https://scalacenter.github.io/scalajs-bundler)
+-----------------------------------------------------------------------
+
+If you want to use Scala.js Bundler (SBT plugin must be enabled in `project/plugins.sbt`) to import jQuery as npm module, then add the following lines to your sbt build definition:
+
+```
+enablePlugins(ScalaJSBundlerPlugin)
+
+libraryDependencies += "be.doeraene" %%% "scalajs-jquery" % "0.9.1"
+
+npmDependencies in Compile ++= Seq(
+  "jquery" -> "2.1.3"
+)
+```
+
+Next define a `jquery` object in your Scala code using the @JSImport annotation to get access to the exported `jQuery` object.
+
+```
+@js.native
+@JSImport("jquery", JSImport.Namespace)
+object jquery extends JQueryStatic
+```
+
+You can access the Scala.js JQuery facade using
+
+```
+package app
+
+import org.scalajs.jquery.JQueryStatic
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSImport
+
+@js.native
+@JSImport("jquery", JSImport.Namespace)
+object jquery extends JQueryStatic
```

Comment:
This is an exact duplicate of the above snippet. It seems redundant and therefore is a bit confusing.